modules/preparing-microshift-executable.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-microshift-executable"]
+= Preparing the MicroShift executable runtime
+:context: creating-microshift-development-environment
+
+Use the following procedure to prepare the MicroShift executable runtime. 
+
+[NOTE]
+====
+When working with MicroShift based on a pre-release minor version `Y` of {OCP}, the corresponding RPM repository `rhocp-4.$Y-for-rhel-8-$ARCH-rpms` will not be available yet. As a workaround, use the `Y-1` version. 
+====
```

Comment:
I'm not sure if we want to say this to customers.

@ggiguash what do you think?